zigpy_znp/api.py

```diff
@@ -203,6 +203,8 @@ async def connect(self, *, test_port=True, check_version=True) -> None:
                 # XXX: Z-Stack locks up if other radios try probing it first.
                 #      Writing the bootloader skip byte a bunch of times (at least 167)
                 #      appears to reset it.
+                LOGGER.debug("Waiting 1s before the skip bytes are sent")
+                await asyncio.sleep(1)
```

Comment:
Thanks! The unit/integration tests rely on calling `ZNP.connect()` a lot so the 1 second delay will make them time out. I think it would be better to move the delay to the end of `uart.connect()`.

Reply:
Moving to delay to the end of uart.connect() still results in many tests failing :/

Reply:
Don't worry about it then, I'll fix the tests.

Reply:
Ok, thanks a lot!